automl/google/cloud/automl_v1beta1/tables/tables_client.py

```diff
@@ -290,6 +292,29 @@ def __model_name_from_args(
             )
         return model_name
 
+    def __log_operation_info(self, message, op):
+        name = "UNKNOWN"
+        try:
+            if (
+                op is not None
+                and op.operation is not None
+                and op.operation.name is not None
+            ):
+                name = op.operation.name
+        except:
+            pass
+        _LOGGER.info(
+            (
+                "Operation '{}' is running in the background. The returned "
+                "Operation '{}' can be used to query or block on the status "
+                "of this operation. Ending your python session with _not_ "
+                "cancel this operation. Read the documentation here:\n\n"
+                "\thttps://google-cloud-python.readthedocs.io/en/stable/core/operation.html\n\n"
```

Comment:
Please use this link: https://googleapis.dev/python/google-api-core/latest/operation.html